mvj/urls.py

```diff
@@ -285,6 +300,7 @@
 
 urlpatterns = [
     path("v1/", include(router.urls + additional_api_paths)),
+    path("pub/", include(pub_router.urls)),
```

Comment:
Hmm, just thinking bout the path.. Should the /pub be under the /v1? (/v1/pub).

Reply:
done